README.md

```diff
@@ -1,59 +1,170 @@
-# Qᴜᴀʟᴛʀᴀɴ
+<!-- H1 title omitted because our logo acts as the title. -->
 
-Qᴜᴀʟᴛʀᴀɴ (quantum algorithms translator) is a set of abstractions for representing quantum 
-programs and a library of quantum algorithms expressed in that language to support quantum 
-algorithms research.
+<div align="center">
+<img alt="Qualtran logo" width="340px" src="docs/_static/qualtran-logo-black.svg#gh-light-mode-only">
+<img alt="Qualtran logo" width="340px" src="docs/_static/qualtran-logo-white.svg#gh-dark-mode-only">
+<br>
 
-**Note:** Qualtran is an experimental preview release. We provide no backwards compatibility 
-guarantees. Some algorithms or library functionality may be incomplete or contain inaccuracies. 
-Open issues or contact the authors with bug reports or feedback.
+Python package for fault-tolerant quantum algorithms research.
 
-Subscribe to [qualtran-announce@googlegroups.com](https://groups.google.com/g/qualtran-announce)
-to receive the latest news and updates!
+[![Licensed under the Apache 2.0 open-source license](https://img.shields.io/badge/License-Apache%202.0-3c60b1.svg?logo=opensourceinitiative\&logoColor=white\&style=flat-square)](https://github.com/quantumlib/qualtran/blob/main/LICENSE)
+[![Compatible with Python versions 3.10 and higher](https://img.shields.io/badge/Python-3.10+-6828b2.svg?style=flat-square&logo=python&logoColor=white)](https://www.python.org/downloads/)
+[![Qualtran project on PyPI](https://img.shields.io/pypi/v/qualtran.svg?logo=python&logoColor=white&label=PyPI&style=flat-square&color=9d3bb8)](https://pypi.org/project/qualtran)
 
-## Documentation
+[Installation](#installation) &ndash;
+[Usage](#usage) &ndash;
+[Documentation](#documentation) &ndash;
+[Community](#community) &ndash;
+[Citation](#citation) &ndash;
+[Contact](#contact)
 
-Documentation is available at https://qualtran.readthedocs.io/
+</div>
+
+Qualtran (_quantum algorithms translator_) is a set of abstractions for representing quantum
```

Comment:
let's remove the "quantum algorithms translator" parenthetical while we're at it. The name "qualtran" isn't an abbreviation for anything. This was trying to explain the joke; which has already been neutered by no longer using the small caps :)

Reply:
I feel like I had a hand in destroying the small caps, which I'm sorry about.

But, honestly, Qualtran = "quantum algorithms translator" wouldn't be bad, would it? Better than many software names I've been guilty of making in the past (e.g., https://github.com/caltechlibrary/baler#readme, https://github.com/caltechlibrary/handprint#readme, etc.)

Reply:
you put small caps out of its dying misery 

abandoning the confusing "is this an abbreviation" halfway through would continue to pay homage to our forefathers

<img width="713" alt="image" src="https://github.com/user-attachments/assets/7c37e159-90b4-4cf9-8a46-53494f92ca52" />

Reply:
Alright, the parenthetical remark is gone in the latest commit.

I also did the last thing I wanted to do, which is improve the way the logo handles dark mode/light mode. Unless I hear otherwise, I'll merge the changes today at 3:00 PST.